lib/features/home/presentation/widgets/full_screen_reel_view.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:video_player/video_player.dart';
+
+class FullScreenReelView extends StatefulWidget {
+  const FullScreenReelView({
+    required this.videoPath,
+    super.key,
+  });
+
+  final String videoPath;
+
+  @override
+  // ignore: library_private_types_in_public_api
+  _FullScreenReelViewState createState() => _FullScreenReelViewState();
+}
+
+class _FullScreenReelViewState extends State<FullScreenReelView>
+    with AutomaticKeepAliveClientMixin {
```

Comment:
This wont help much @ritesh-chaudhari-aepl as this is the full screen itself rather than the item. Unecessary to keep it here
```suggestion
    with AutomaticKeepAliveClientMixin {
```